docs/framework/wcf/samples/udp-activation.md

```diff
@@ -71,7 +71,7 @@ This sample is based on the [Transport: UDP](../../../../docs/framework/wcf/samp
   
 1. Install ASP.NET 4.0 using the following command.  
```

Comment:
I have commented in another PR.  In .NET 4.x above, the way to enable ASP.NET is to enable the components in the UI, or running dism command.  Could we simplify this by just saying enable ASP.NET component?

Reply:
As I mentioned, the requested task for the vendor team was to add missing language identifiers throughout the docs repo.
If content needs update, it would probably better if you did a search and replace of this sentence throughout the repo @HongGit and we can help merge.

Thanks!